lib/find-python.js

```diff
@@ -435,27 +435,26 @@ class PythonFinder {
         )
         this.log.silly('execFile: opts = ', JSON.stringify(opts, null, 2))
 
-        //* prosible outcomes with error messages on windows (err.message, error.stack?, stderr)
+        //* possible outcomes with error messages on Windows (err.message, error.stack?, stderr)
         // issue of encoding (garbage in terminal ) when 866 or any other locale code
         // page is setted
         // possible solutions:
-        // 1. let it be as it is and just warn user that it should use utf8
-        // (already done in this.catchErros's info statement)
-        // 2. somehow determine user's terminal encdoing and use utils.TextDecoder
-        // with getting raw buffer from execFile.
-        // Requires to correct error.message becouse garbage persists there
-        // 3. Forse user's terminal to use utf8 encoding via e.g. run "chcp 65001". May broke some user's programs
+        // 1. leave it as is and just warn the user that it should use utf8
+        // (already done in this.catchError's info statement)
+        // 2. somehow determine then user's terminal encoding and use utils.TextDecoder

```

Comment:
```suggestion
        // 2. somehow determine the user's terminal encoding and use utils.TextDecoder

```

Reply:
can i use `git commit --amend` and force push or it would be better to just make new commit?

Reply:
Do as you see fit.

Below, it says `Conflicting files gyp/pylib/gyp/input.py` so you will need to rebase.

Reply:
Some test are not passing because in `gyp/pylib/gyp/input.py` on line `228` in call `open(build_file_path)` should be `encoding='utf-8'`. I just cant add it ther due merge conflict.